speech/grpc/README.md

```diff
@@ -73,20 +73,35 @@ note that the audio file must be in RAW format.  You can use `sox`
 (available, e.g. via [http://sox.sourceforge.net/](http://sox.sourceforge.net/)
 or [homebrew](http://brew.sh/)) to convert audio files to raw format.
 
-### Run the non-streaming client
+### Run the sync client
```

Comment:
@lesv we don't have the instructions anywhere otherwise and assuming users would have read everything before using this sample is not correct either. So we need them IMHO.